pages/Javascript-API.md

```diff
@@ -121,6 +112,43 @@ Re-parses the `scene.config` object and redraws the scene, updating data sources
 scene.updateConfig()
 ```
 
+## Events
+
+Tangram provides two selection event handlers: `hover` and `click`, and one render state event emitter: `view_complete`.
+
+#### `hover` and `click`
```

Comment:
These are definitely the best/easiest way to access feature picking now, so we may want to reconfigure this to come before the [`getFeatureAtPixel()`](https://github.com/tangrams/tangram-docs/pull/77/files#diff-baa1f32fc3290cda18e6b528f174ffb5L49) section -- or at least find a way to make it clear that these events are the preferred way to get feature info.